plugins/source/alicloud/config/alicloud.yaml

```diff
@@ -0,0 +1,22 @@
+kind: source
+spec:
+  name: "alicloud"
+  registry: "local"
+  path: "cloudquery-alicloud"
+  destinations: ["csv"]
+  tables:
+    - alicloud_oss_*
+    - alicloud_bss_*
+  spec:
+    region_id: ""
+    access_key: ""
+    secret_key: ""
+---
+kind: destination
+spec:
+  name: "csv"
+  path: "cloudquery/csv"
+  version: "v1.1.12"
+  write_mode: "append" # CSV only supports 'append' mode
+  spec:
+    directory: './cq_csv_output'
```

Comment:
This is a good way for developers to test the plugin against real infrastructure for now, but eventually we'll probably remove it here and move it to the docs. 

For testing in CI we'll require mock tests for the resolvers. Here's [an example from the Github plugin](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/github/resources/services/external/groups_test.go). I can also help you write those if you like, they are fairly mechanical once you've created the mocks :)